### PR TITLE
Feature/data

### PR DESCRIPTION
**Pull request by Elgin-Schultz**

### CODE REVIEW

File: components/GlobalStyles.js [hunk 1]
```diff
@@ -1,16 +1,57 @@
...
+        padding: 0;
+        font-family: 'Trade Gothic';
     }
+    body {
+        
 
+        
+        margin: 0;
+        padding: 10%;
+
```
Comment (julialoeschel) [inline components/GlobalStyles.js:33]: please check for blank lines - keep the structure for better readability

File: components/SitesList.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import styled from "styled-components";
+import { allSites } from "../lib/sites";
+
+export default function SitesList() {
+  const sites = allSites.map((site) => {
+    return (
+      <li>
+        <mark>{site.pos[0] + " | " + site.pos[1]}</mark>
```
Comment (julialoeschel) [inline components/SitesList.js:8]: If you want to apply a backgrond color for all Items, please to that with CSS. The <mark> is only for Highligting.

File: lib/sites.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
+export const allSites = [
+  {
+    key: 1,
+    pos: [53.577587, 9.991238],
```
Comment (julialoeschel) [inline lib/sites.js:4]: the common name for such data would be `coordinates`. 
if you want to keep it, i recommand writing the whole name for better readability.

File: pages/index.js [hunk 4]
```diff
@@ -1,3 +1,18 @@
...
+
 export default function Home() {
-  return <h1>Hello Beavers 🦫</h1>;
+  return (
+    <>
+      <Head>
+        <title>Profan</title>
+      </Head>
+      <h1>
+        pro|f<u>a</u>n
```
Comment (julialoeschel) [inline pages/index.js:12]: cool that you found this, but mdn recommends not to use this for such a purpose: 
_You should not use <u> to underline text for presentation purposes, or to denote titles of books._

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/u?retiredLocale=de
maybe we could use a span and tyole the underline here?

**Review by julialoeschel — COMMENTED** [review]
Great work there 🐜 

I only have a view comments

keep on going!!

File: lib/sites.js (hunk 3 above)
Comment (Elgin-Schultz) [reply lib/sites.js:4]: I will change to coordinates!

File: pages/index.js (hunk 4 above)
Comment (Elgin-Schultz) [reply pages/index.js:12]: Could not find tyole... What does that mean?

File: components/SitesList.js (hunk 2 above)
Comment (Elgin-Schultz) [reply components/SitesList.js:8]: Have te problem with block element when using CSS. But will try again... As I am not sure if I am going to keep the list optic, I might not want to use so much time on that for now ...

File: pages/index.js (hunk 4 above)
Comment (julialoeschel) [reply pages/index.js:12]: OH, that went wrong. span was the word I was looking for.

**Pull request merged**